### PR TITLE
Update EvacTeamMember/VisitNote Ui

### PR DESCRIPTION
**Pull request by trevorskaggs**
- [x] Update New EvacTeamMember UI > Use Cards, Buttons On Bottom
- [x] Update VisitNote UI > Buttons on Bottom
- [x] Add Display Formatting for Phone Fields

### CODE REVIEW

**Comment by trevorskaggs** [comment]
Ready for review! Big stuff is highlighted up top, otherwise small UI tweaks!

File: frontend/src/evac/EvacForms.js [hunk 1]
```diff
@@ -25,12 +24,12 @@ import Header from '../components/Header';
 import { Checkbox, DateTimePicker, DropDown, TextInput } from '.././components/Form';
 import { statusChoices } from '../animals/constants';
 
-export const EvacTeamMemberForm = () => {
+export const EvacTeamMemberForm = ({id}) => {
```
Comment (AlexMountain) [inline frontend/src/evac/EvacForms.js:27]: Why did we add `id` here?  As far as I can tell, this form can only be used to create new team members, and we have no way to update/edit them at present.

File: frontend/src/evac/EvacForms.js [hunk 2]
```diff
@@ -71,54 +70,58 @@ export const EvacTeamMemberForm = () => {
...
-                      label="Agency ID"
-                      name="agency_id"
-                      id="agency_id"
-                    />
-                  </Col>
-                </Row>
-              </FormGroup>
-              <Button type="button" className="btn btn-success mr-1" onClick={() => {setAddAnother(false); form.submitForm()}}>Save</Button>
+          <Card border="secondary" className="mt-5">
+            <Card.Header as="h5" className="pl-3"><span style={{cursor:'pointer'}} onClick={() => window.history.back()} className="mr-3"><FontAwesomeIcon icon={faArrowAltCircleLeft} size="lg" inverse /></span>{!id ? "New" : "Update"} Team Member</Card.Header>
```
Comment (AlexMountain) [inline frontend/src/evac/EvacForms.js:74]: This seems like our only use of `id`?  As mentioned before, this seems unnecessary without more functionality.

File: frontend/src/router.js [hunk 3]
```diff
@@ -27,8 +26,8 @@ const routes = {
   "/animals/search": () => <AnimalSearch />,
   "/animals/animal/:id": ({id}) => <AnimalDetail id={id} />,
   "/evac": () => <Evac />,
+  "/evac/evacteammember/:id": ({id}) => <EvacTeamMemberForm id={id} />,
   "/evac/evacuationassignment/search": () => <EvacuationAssignmentSearch />,
-  "/evac/evacteammember/new": () => <NewEvacTeamMember />,
```
Comment (AlexMountain) [inline frontend/src/router.js:31]: I think we should revert this and continue to use the `/new` url when creating a new member.

**Review by AlexMountain — COMMENTED** [review]
Forms are lookin good, phone numbers are looking sexy, and I dig the other changes.   I think we just need to figure out what's going on with the `id` inclusion to the New Team Member Form.

**Comment by trevorskaggs** [comment]
This ticket should be ready again!

**Review by AlexMountain — APPROVED** [review]
@trevorskaggs Good to merge!

**Pull request merged**